### ISSUE TITLE
[UX] 10-10EZ Toxic Exposure Research Follow-Up Items

### ISSUE THREAD

**Issue by aagosto90** [department-of-veterans-affairs/va.gov-team#81905]
## Issue Description
Now that research synthesis for 10-10EZ TERA questions is complete and we've identified next steps as team, the UX team will move forward on suggested changes.

- [Slack thread with team follow-up items](https://dsva.slack.com/archives/CMJ2V70UV/p1714151834266889)
- [Research folder](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/health-care/application/va-application/research/2024-04-ToxicExposure-UsabilityStudy)

---
## Tasks
- [x] Open up convo about form scrolling issue again (Angela)
- [x] Sync up with VES team on a couple items: when their system will be able to accept special characters in free text field for other toxic exposures and more importantly, find out if there are certain required data inputs for TERA questions that we'll need to consider when proposing new branching logic in the form
- [x] Make a low priority ticket for the backlog for the Read Aloud screen reader issue (Angela)
- [x] Make a ticket to cover TERA content updates for both UX and CAIA (Angela)
- [x] Prep for HEC meeting on 5/8 to share research findings and recommendations with stakeholders

## Acceptance Criteria
- All tasks above have been completed and UX is ready to start working on content updates in a subsequent ticket
---

**Comment by aagosto90** [comment]
[Ticket that DK opened about scrolling issue here](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2124)
[Slack message to resurface this problem here](https://dsva.slack.com/archives/C01DBGX4P45/p1714488207826619)

TLDR: Matt Dingee knows ownership of the forms library is confusing and hopes OCTO will have a better solution for this soon. In the meantime, the DST is heads down on v3 component updates and will have to prioritize issues like this after that.

**Comment by aagosto90** [comment]
Heather confirmed with VES that they will be able to accept special characters in the other toxic exposures field as of 6/10/24. [Slack message here](https://dsva.slack.com/archives/CMJ2V70UV/p1714666977081299). So we should plan to remove the explainer text on that question when that functionality goes live. I'll create a FE ticket for the backlog to update this after 6/10.
